docs/streams.html

```diff
@@ -398,6 +401,84 @@
         in <code>StreamsConfig</code> before running it. A complete list of
         Kafka Streams configs can be found <a href="#streamsconfigs"><b>here</b></a>.
         </p>
+
+        <h2><a id="streams_upgrade" href="#upgrade">Upgrade guide and API changes</a></h2>
+
+        <h3><a id="streams_upgrade_1020" href="#upgrade_1020">Upgrading a Kafka Streams Application</a></h3>
+
+        <h4>Upgrading from 0.10.1.x to 0.10.2.0</h4>
+        <p>
+        See <a href="../#upgrade_1020_streams">Upgrade Section</a> for details.
```

Comment:
The link name here "Upgrade Section" (for anchor `#upgrade_1020_stream`) does not match the name of the target "Upgrading a Kafka Streams Application".